src/poisson_dev_fe.jl

```diff
@@ -571,6 +571,29 @@ low_level_manual_gradient_dv_array = lazy_map(Broadcasting(Operation(⋅)),inv_J
 
 # ## A low-level implementation of the residual integration and assembly
 
+# The discretized variational form of the Poisson problem reads as:
+# >Find $u_h \in V_h^{\Gamma_D}=\{v_h \in V_h:v_h=g_0 \; on \; \Gamma_D\}$ such that
+# >```math
+# >(u_h,v_h)=l(v_h)\quad \forall v_h \in V_h^0
+# >```
+# >where $V_h^0=\{v_h \in V_h:v_h=0 \; on \; \Gamma_D\}$.
+#
+# Take $u_h = \hat{u}_h-w_h$ where $w_h \in V_h^0$ and $\hat{u}_h \in V_h^{\Gamma_D}$, then
```

Comment:
I would rewrite this sentence as follows. 

```
If we take an arbitrary function $\hat{u}_h in V_h^{\Gamma_D}$, and 
compute $w_h \in V_h^0$ as the solution of the following discrete variational problem: 

Then, it is easy to see that `$u_h = \hat{u}_h-w_h$`. This is the strategy 
that we implement in the sequel. Note that, in a previous section, 
we created a FE function using `rand` for initializing the values of the 
degrees of freedom. This function indeed plays the role of \hat{u}_h.
```
What do you think?

Reply:
definitely better than mine